requirements.txt

```
@@ -4,6 +4,7 @@ ray >= 2.9
 sentencepiece  # Required for LLaMA tokenizer.
 numpy
 torch == 2.1.2
+tensorizer >= 2.8.0
```

Force people to update their tensorizer versions more! (When there are no breaking changes)
```suggestion
tensorizer >= 2.8.1
```